spec/unit/pool_manager_spec.rb

```diff
@@ -2704,6 +2774,7 @@
       allow(subject).to receive(:check_snapshot_queue)
       allow(subject).to receive(:check_pool)
       allow(subject).to receive(:check_ondemand_requests)
+      allow(subject).to receive(:get_domain_for_pool).with(pool).and_return('example.com')
```

Comment:
I think the spec error is due to this with(), it could be removed to accept any parameter.

Reply:
Nice catch! That did indeed fix the tests